core/src/main/java/org/arquillian/smart/testing/configuration/ObjectMapper.java

```diff
@@ -69,9 +70,23 @@ private static Object getConvertedObject(Method method, Object configFileValue,
             Object mappedValue = null;
             ConfigurationItem configItem = foundConfigItem.get();
 
-            if (configItem.getSystemProperty() != null) {
+            if (configItem.getSystemProperty() != null && !configItem.getSystemProperty().endsWith("*")) {
                 mappedValue = System.getProperty(configItem.getSystemProperty());
             }
+
+            if (configItem.getSystemProperty() != null && configItem.getSystemProperty().endsWith("*")) {
```

Comment:
Can we check `configItem.getSystemProperty().endsWith(".*")`? to avoid getting wrong propertyName using  `configItem.getSystemProperty().substring(0, configItem.getSystemProperty().lastIndexOf('.'));`

Otherwise something will be wrong for this type of input `smart.testing.strategy.custom*`

Reply:
I thought the same as you, but for any reason that now I cannot remember I decided to use `*`. I change it because now seems more reasonable what you mention.